### DISCUSSION TITLE
如何将uno.config.ts中定义的所有原子样式都应用到项目中

### DISCUSSION THREAD

**Discussion by LearningUnity** [unocss/unocss#4203] [Q&A]
大佬们, 我在做一个工具, 需要动态使用原子样式, 比如项目脚本内没有用到 h-full, 但是有逻辑会动态添加 h-full, 这时候要怎么处理呀, 可以通过配置解决吗

**Comment by zyyv** [comment]
@LearningUnity UnoCSS is extracted on demand, if you want to apply some atomic classes in advance, you can try `safelist`.
```ts
// uno.config.ts

{
  safelist: ['h-full']
}
```
